changelogs/current.yaml

```
@@ -330,6 +330,10 @@ new_features:
   change: |
     added stat ``overload.refresh_interval_delay`` to track the delay between overload manager resource loop refresh in
     milliseconds.
+- area: load shed point
+  change: |
+    added load shed point ``core.listener_accept`` that rejects new tcp connections
+    by closing the connection at the during the new connection accept phase.
```

```suggestion
    by closing the connection during the new connection accept phase.
```
BTW: it may be better to state "before the new connection accept phase", as the connection is not accepted IIUC.

---

done.